components/script/dom/htmlmediaelement.rs

```diff
@@ -1263,6 +1266,11 @@ impl HTMLMediaElementMethods for HTMLMediaElement {
     // https://html.spec.whatwg.org/multipage/#dom-media-autoplay
     make_bool_setter!(SetAutoplay, "autoplay");
 
+    // https://html.spec.whatwg.org/multipage/#dom-media-defaultmuted
+    make_bool_getter!(DefaultMuted, "muted");
```

Comment:
The `make_bool_getter!` and `make_bool_setter!` macros implement the getter and setter of HTML attributes. (You can see its implementation [here](https://github.com/servo/servo/blob/f1dd31f70440fa9c7a40525bd1e03eede568f74d/components/script/dom/macros.rs#L18). In this case, `defaultMuted` is not a HTML attribute of the `<video>` or `<audio>` elements, but a property of the `HTMLMediaElement`JS object, so we don't need to use these macros here. We have to implement the getter and setter manually, just like you did with `muted`.